src/globals.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * globals.ts hold some globals used throughout the extension
+ */
+
+// true for running tests, false during regular runtime
+declare var isTesting: boolean;
```

Comment:
Can you use let here? 

Also the best way to do this is:

`export let isTesting = false;`

(though actually export the object, not just the boolean!